apps/etterlatte-beregning-kafka/src/main/kotlin/no/nav/etterlatte/beregningkafka/MigreringBeregningHendelserRiver.kt

```diff
@@ -69,18 +69,9 @@ private fun verifiserNyBeregning(
     }
 
     with(beregning.beregningsperioder.first()) {
-        check(grunnbelop == migreringRequest.beregning.g) {
-            "Beregning må være basert på samme G som i Pesys. Er $grunnbelop i Gjenny, " +
-                "var ${migreringRequest.beregning.g} i Pesys."
-        }
-        check(utbetaltBeloep >= migreringRequest.beregning.brutto) {
```

Comment:
Hmm, gir det meining å ta bort denne? Fordi det har skjedd ting i livet som gjer at søkar no får mindre enn før, kanskje?

Reply:
Jeg er ikke sikker men vi skal nok ikke bare sette det samme som før hvertfall. Mulig vi kan sende dent til manuell.
Men jeg tror det gir mening å tenke som du sier. Det kan ha skjedd ting siden sist og det er ikke en løpende ytelse lenger.
jeg dobbeltsjekker med Helge.

Reply:
Det eneste som reelt sett påvirker ytelsen vår er jo trygdetid (evt. endring i familiebilde, men der har vi sett at det ikke er noen varianter hvor bruker kommer dårligere ut), så hvis de kommer dårligere ut på ny ytelse nå så er det noe veldig rart, og jeg gjetter på at vi bør sende den til manuell.

Reply:
Svar fra Helge:
Ja, det garantien mener jeg ikke gjelder for disse sakene. I utgangspunktet tenker det var en unødvendig sjekk også for BP < 18 år, siden det kun var de foreldreløse dette kunne gjelde for (og de ble tatt manuelt med overstyrt beregning)

Jeg legger den uansett inn igjen. Det bør ikke skje om om det skulle skje av en rar grunn så sender vi den til manuelt.